Polyhedron/demo/Polyhedron/Plugins/PMP/Orient_soup_plugin.cpp

```diff
@@ -271,5 +295,141 @@ void Polyhedron_demo_orient_soup_plugin::displayNonManifoldEdges()
     QApplication::restoreOverrideCursor();
   }
 }
+void Polyhedron_demo_orient_soup_plugin::createPointsAndPolyline()
+{
+  
+  typedef std::pair<std::size_t, std::vector<std::size_t> > V_ID_and_P_IDs;
```

Comment:
Lots of warnings:
```
Plugins/PMP/Orient_soup_plugin.cpp:291:61: warning: typedef 'V_ID_and_P_IDs' locally defined but not used [-Wunused-local-typedefs]
```
Any linux developer should see those warnings. You must work with flags `-Wall -Wextra`. You can add `-Wno-maybe-uninitialized` to clean-up the warnings output when you compile in Release.